go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/imkira/go-interpol v1.1.0 // indirect
+	github.com/klauspost/pgzip v1.2.5
```

Comment:
licence?

Reply:
It's MIT so should be fine: https://github.com/klauspost/pgzip/blob/master/LICENSE

Is there anything else we should do regarding license?